shopify/wishlist-template/extensions/wishlists/src/components/NewWishlist.tsx

```diff
@@ -50,8 +64,8 @@ export default ({ wishlists }) => {
             render={({ field: { ref, ...fieldProps } }) => (
               <TextField
                 label="Name"
-                onChange={(text) => fieldProps.onChange(text)}
-                {...{ fieldProps }}
+                {...fieldProps}
+                onChange={(text: string) => fieldProps.onChange(text)}
```

Comment:
Do we need to manage this manually? I think removing this line is okay

Reply:
I might be able to remove it. I think it caused issues last time

Reply:
Fixed